apps/docs-website/docs/requirements/recommendations.md

```diff
@@ -6,6 +6,8 @@ description: You probably need this too
 # Recommendations
 
 - [Node >=22](https://nodejs.org/en/)
+- [ESLint >=9.0.0](https://eslint.org/)
+- [Typescript >= 5.0.0](https://www.typescriptlang.org/)
```

Comment:
These are already listed in the minimums. I think we do want to recommend newer versions though, perhaps this?:

```suggestion
- [ESLint >=9.5.0](https://eslint.org/)
- [Typescript >= 5.7.0](https://www.typescriptlang.org/)
```

b/c 9.5 introduced language plugins and is the version ESLint tooling depends on, and 5.7 b/c it's needed for `@types/node@>=22.7.3`.

Reply:
Also, I think this will conflict with #305.

Reply:
i basically "duplicated" here these requirements because it felt odd not having the hard requirements as a base for the recommended requirements. I mean, they **are** intended as a base of course, but making it explicit avoid any confusion i think.